CryptoBook-py/CryptoBook/handlers.py

```diff
@@ -127,19 +148,28 @@
     # Check to see if the symbol is available on the exchange.
     if symbol not in ex.symbols:
         return (
-            False,
-            400,
             {
                 "error": "symbol_error",
                 "description": "The requested symbol ({}) is not available from {}.".format(
                     symbol, exchange
                 ),
                 "symbols": ex.symbols,
             },
+            400,
         )
 
     # Closing connection with the market.
     await ex.close()
 
+    # Attempts to get the request.
+    try:
```

Comment:
I see that you use a similar error handling statement above in `ip(...)`. Perhaps consider grouping the error handling statements in `ip()` and `historical_data()` into a single `try_request(...)` method a la: 

```
def try_request(func, *args, **kwargs):
  try:
    resp = await func(*args, **kwargs)
  except ExchangeDataAccuracyError as e:
    return {"error": "exchange_data_accuracy_error", "description": str(e)}, 400
  except NetworkError as e:  # pragma: no cover
    return {"error": "network_error", "description": str(e)}, 400
  except Exception as e:  # pragma: no cover
    return {"error": "server_error", "description": str(e)}, 400 
```
so `ip()` now invokes:
```
resp = await try_request(get_exchange_info, exchange=exchange)
```

and `historical_data()` now invokes:
```
resp = await try_request(get_historical_data, **request)
```

Reply:
I like this idea. Will change the codebase!

Reply:
Added on [cb8ad5a](https://github.com/Waultics/CryptoBook/pull/30/commits/cb8ad5a366710fa63099b35755b2b53123c91112). Might have to add a few tests for coverage not dip under 100%.

Reply:
Removed coverage on function `util:handle` on [23b2895](https://github.com/Waultics/CryptoBook/pull/30/commits/23b289592042849c68610279d6adc9cf97f8a24e).